src/platform/graphics/graphics-device.js

```diff
@@ -714,6 +732,31 @@ class GraphicsDevice extends EventHandler {
         this.vertexBuffers.length = 0;
     }
 
+    /**
+     * Retrieves the slot in internal {@link StorageBuffer}, which is used for indirect rendering.
+     * This slot can be used in by a {@link Compute} to generate indirect draw parameters, and by
+     * setting up indirect drawing by using {@link MeshInstance#setIndirect}.
+     *
+     * @returns {number} - The slot used for indirect rendering.
+     */
+    getIndirectDrawSlot() {
+        return 0;
+    }
+
+    /**
+     * Returns the buffer used to store arguments for indirect draw calls. The size of the buffer is
+     * controlled by the {@link GraphicsDevice#maxIndirectDrawCount} property. This buffer can be
+     * passed to {@link Compute} shader along with a slot obtained by calling
+     * {@link GraphicsDevice#getIndirectDrawSlot}, in order to prepare indirect draw parameters.
+     * Also see {@link MeshInstance#setIndirect}.
+     * Only available on WebGPU, returns null on other platforms.
```

Comment:
Either follow on from the line above or add another newline if your intention was to start a new paragraph. Check the JSDoc output.